docs/en/create_connection/mssql.md

```diff
@@ -0,0 +1,30 @@
+# Connecting MsSQL
+
+To configure [Yii Db Mssql](https://github.com/yiisoft/db-mssql) without [di container](https://github.com/yiisoft/di), you need to follow the following steps.
```

Comment:
I think it may be merged into the same file that contains DI container config example.